pkg/policies/controlplane/runtime/circuit_test.go

```diff
@@ -83,3 +83,62 @@ var _ = Describe("Circuit", func() {
 		))
 	})
 })
+
+var _ = Describe("Integrator", func() {
+	It("integrates signals", func() {
+		circuit, err := sim.NewCircuitFromYaml(
+			`
+      evaluation_interval: 1s
+      components:
+      - integrator:
```

Comment:
Merge latest main. I changed integrator a bit.

Reply:
Also, integrator had lot of bugs -- e.g. min/max not doing what was supposed to, absence of min or max would give wrong result and so on. I fixed some bugs yesterday and fixing more today. 
As a learning, we need our test cases to cover these corner cases as this PR could have caught many issues as well. cc: @DariaKunoichi

Reply:
@harjotgill thanks for catching it! I'll work on adding more tests in my future tasks